### ISSUE TITLE
model.train() isn't called again after model.eval()

### ISSUE THREAD

**Issue by beegica** [tk-rusch/coRNN#1]
https://github.com/tk-rusch/coRNN/blob/ba26cee3a74c46ba4ed3a2dc73249efe39f80827/adding/adding_task.py#L45

I think in this task you forget to turn training back on after doing a test. You call model.eval() on line 45, but then model.train() doesn't get called again. I am currently testing this code, and will report back if it is unable to learn the adding problem. I am sure it is just a small error though and that a different training script was run while writing the paper!

Let me know if I am mistaken, thanks!

**Comment by tk-rusch** [comment]
Hi, thanks for your comment. Please note that this does not affect the outcome of the model, as model.eval() and model.train() changes the model behavior only if something like dropout or BatchNorm is used (it changes the self.training to True or False in the class "coRNN"). Since we don't use anything of that in our model, it doesn't affect the training. 
I will, however, change the github code accordingly in order to prevent any confusion.
